pdr-api/handlers/fees/GET/index.js

```diff
@@ -0,0 +1,150 @@
+const { runQuery, runScan, TABLE_NAME} = require('/opt/dynamodb');
+const { sendResponse, logger } = require('/opt/base');
+
+
+exports.handler = async (event, context) => {
+
+  logger.debug('Get all fees for specific park', event);
+  if (event.httpMethod === 'OPTIONS') {
+    return sendResponse(200, {}, 'Success', null, context);
+  }
+
+  try {
+    const queryParams = event.queryStringParameters;
+    const queryType = getQueryType(queryParams);
+
+    let query;
+    if(queryParams){
+      switch (queryType) {
+        case 'withBilling':
+          if (!queryParams.billingBy || !queryParams.activity || !queryParams.parkFeature || !queryParams.ORCS) {
+            throw {
+              code: 400,
+              error: 'Insufficient parameters.',
+              msg: `Missing required parameters for 'withBilling' query`
+            };
+          }
+          query = queryFeeWithBilling(queryParams.billingBy, queryParams.activity, queryParams.parkFeature, queryParams.ORCS);
+          break;
```

Comment:
Since the entire switch statement is getting large, would prefer to see something wrapping all functionality like:

SOME_VARIABLE='someVariable'`
```
case WITH_BILLING:
  query = handleFeeWithBilling(queryParams);
  break;
case BY_ACTIVITY:
  query = handleFeeByActivity(queryParams);
  break;
...
```

and inside the `handleFeeWithBilling` just throw instead of checking and throwing before you call the function.

const's declared above, and functions to handle each case. Makes UT's easier to isolate and much easier to read. Also throughout the code avoid repeating the string 'someVariable' and instead use a `const

Reply:
That makes sense, Re worked in : [Commit](79210c7e569ed7e600615bc738cc5889c18bc619) 

If that's not what you had in mind let me know!